invenio_app_ils/ill/schemas/ill_libraries/library-v1.0.0.json

```
@@ -12,6 +12,10 @@
       "title": "The email of the external library",
       "type": "string"
     },
+    "legacy_id": {
+      "type": "string",
+      "title": "The previous id of this Library in CDS before migration"
```

Please remove CDS from here (InvenioILS) :) Check also the description of the other `legacy_id` field if we put by mistake.
```suggestion
      "title": "The previous id of this Library before data migration"
```